net/reaver/patches/100-nl80211-channel-switching.patch

```diff
@@ -0,0 +1,72 @@
+--- a/src/iface.c
++++ b/src/iface.c
+@@ -34,6 +34,10 @@
```

Comment:
> Upstream code resolved their issue in 2020 for this without fixing, hence this patch.

Why downstream should patch it? It does not make any sense. There isn't any Git header.
You are not author of this changes, so you should here add commit header, description, author, etc.

Reply:
Upstream won't fix, see https://github.com/t6x/reaver-wps-fork-t6x/issues/324

I just grabbed the code from one of my other tools that I have authored. I can add a header stating that I'm the author for the channel switch code and add a header for the aircrack-ng channel to frequency code.

Reply:
It's ``OpenWrt`` not ``OpenWRT``.

- Exactly as you do, I don't see a reason that we should merge this. We should rather remove this package once and for all. How many users have installed this package on their router (or rather, how many times it was downloaded)? Wireless extensions are disabled as you said in the kernel 5.15 and very few drivers use it.

From the issue which they won't fix:
> disabling WPS so reaver may already have become useless once the old wireless interface has been removed from the kernel.

We removed it (= disabled) from the kernel at least for OpenWrt. If you have your own usage, it can be compiled locally, isn't it?

- Upstream does not fix it, so why should we? Who is going to review it? Who will take care of issues, which it could bring?

- Of course, if you authored it, you should add it to the patch itself.

It is interesting that Ubuntu and as well Debian do not patch it.

This is up to @yousong to decide what we are going to do with this package as he imported it from oldpackages.